engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +231,44 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float function1(float parameter){
+        double p = (double)parameter;
+        return (float) pow(p,0.6) * 0.78f ;
```

Comment:
What's the reason to cast between float and doubles here? Wouldn't the following work?

```java
private double calculateSomething(double p) {
    return pow(p, 0.6) * 0.78;
}
```

I guess this could even be inlined.

Reply:
The values returned from the functions are assigned to float variables.Hence the cast